onnx/model_container.py

```diff
@@ -102,10 +110,16 @@ def enumerate_graph_protos(self) -> Iterable[onnx.GraphProto]:
         yield self.model_proto.graph
         yield from _enumerate_subgraphs(self.model_proto.graph)
 
+    def is_in_memory_external_initializer(self, name: str) -> bool:
```

Comment:
This method does not use self. Move to global name space as a public function?

Reply:
I prefer to keep it in the class since it cannot be applied for anything else. I hesitated with a class method. I chose a method to be able to add more code using the existing names later.

Reply:
Ok. Thanks!